lib/board.rb

```diff
@@ -0,0 +1,52 @@
+require 'pry'
+class Board
+  attr_accessor :cells, :token
+
+
+
+  def reset!
+   @cells = Array.new(9, " ")
+  end
+
+  def initialize
```

Comment:
It is customary to declare `initialize` before any other instance methods in a class.